src/integer/z/arithmetic/add.rs

```diff
@@ -21,6 +22,9 @@ use flint_sys::{
     fmpq::fmpq_add_fmpz,
     fmpz::{fmpz, fmpz_add},
     fmpz_mod::fmpz_mod_add_fmpz,
+    fmpz_poly::{
+        fmpz_poly_get_coeff_fmpz, fmpz_poly_set, fmpz_poly_set_coeff_fmpz, fmpz_poly_struct,
+    },
```

Comment:
probably dont need all when adapting the changes

Reply:
cleared